paseos/paseos.py

```diff
@@ -95,7 +95,6 @@ def add_known_actor(self, actor: BaseActor):
     @property
     def local_actor(self) -> BaseActor:
         """Returns the local actor.
-
```

Comment:
Please revert these white space changes. There should be an empty line after description to follow https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html